templates/message/filter.html.twig

```diff
@@ -10,11 +10,7 @@
     <span class="text--bold text--black">2</span> / 2
 {% endblock %}
 
-{% block message_content %}
-    {% include('message/filter/'~ message.type ~'.html.twig') %}
-
-    <div id="message-actions-block" class="l__wrapper--slim background--lighter b__nudge--top-40 text--center"></div>
```

Comment:
can you keep this div here, because it's common div for all filter page and it's going with `App.runMessageFilters(...)` javascript code.
you can declare a new block `filter_content` for example